src/test/regress/sql/columnar_chunk_filtering.sql

```diff
@@ -445,3 +445,15 @@ RESET columnar.max_custom_scan_paths;
 RESET columnar.qual_pushdown_correlation_threshold;
 RESET columnar.planner_debug_level;
 DROP TABLE pushdown_test;
+
+--
+-- https://github.com/citusdata/citus/issues/5803
+--
+
+create table clause_check (id int, company_name text) using columnar;
+INSERT INTO clause_check VALUES(1, 'AAPL');
+INSERT INTO clause_check VALUES(2, 'IBM');
+INSERT INTO clause_check VALUES(3, 'TSLA');
```

Comment:
Sorry for the nitpick, but I don't think it's a good idea to include other company's ticker symbols in our code base. Not sure *why* it would be bad, but it just seems off. Let's use states or countries or something?